### ISSUE TITLE
Show helpful error message when login fails due to unexpected response from idp

### ISSUE THREAD

**Issue by Wladefant** [joschahenningsen/TUM-Live#1159]
**To Reproduce**
1. Tap login
2. Tap login with tum
3. Enter your login information
4. Tap continue (or something similar)
5. you are redirected to https://tum.live/shib that is a dead link (at least for now 
![image](https://github.com/TUM-Dev/gocast/assets/79611093/436970f5-6920-4c89-8330-7820110ffbae)

- Windows 11
- Microsoft Edge

**Comment by joschahenningsen** [comment]
A few quick questions to help me find the issue: 

- Have you ever logged into tum live?
- Are you a student, employee or guest?
- Will you start your studies this upcoming semester/is your TUM account relatively new?
- What's your tum-id? (e.g. ge42tum)

Thanks for your help!

**Comment by Wladefant** [comment]
1. No
2. Student
3. Yes (not even immatrikulsted at the momet)
4. go69ziq

**Comment by joschahenningsen** [comment]
I suppose the reason for this is, that your account is not fully active yet
untill October. Can you try using the internal login with your normal tum
credentials and let me know if it works then?

**Comment by Wladefant** [comment]
Yes that works thank you, but that still needs to be solved more gracefully 😊

**Comment by joschahenningsen** [comment]
Definitely! I changed the title accordingly.
